src/components/WeatherView/WeatherStories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import WeatherView from './WeatherView.js';
+import { belfastWeather } from '../../routes/Weather/modules/WeatherHelper.js';
```

Comment:
Can this WeatherHelper.js file be moved into the /WeatherView/ folder and be renamed to WeatherFixtures.js ?

It's not a module in the Redux sense. A fixture is just a piece of stub data, so should be appropriate for this.

That said, there is a lot more data in the fixture than the View here needs. I know our requirements are a little backwards here (We're looking at an API and coming up with something to build), but really, we should be starting with what's on screen and working out what data we need to do that.

With that approach, the View component should be explicit about what props it expects. Instead of taking a full API response, it should take individual properties for each piece of data.

A reducer should then map from the API response format to the flattened, limited amount that the view needs.

Check out my RepoList as an example. In RepoListDucks.js, you can see the LOAD_SUCCESS reducer doing that mapping.

It's just another means of removing logic from your Views. It shouldn't need to understand the structure of the particular API we use. It should just care that its sunny and 21 degrees :)